lib/portage/util/__init__.py

```diff
@@ -2006,3 +2008,11 @@ def read_ld_so_conf(path):
     rval.append("/lib")
 
     return [normalize_path(x) for x in rval if x]
+
+
+def no_color(nocolor) -> bool:
```

Comment:
I was expecting a comment here to say we can clean up NOCOLOR support everywhere in future.

Also, 'nocolor' is not a good variable name here, it should be settings (with a type annotation)?